client/overview/task-list/tasks.js

```diff
@@ -88,7 +106,7 @@ export const getTasks = ( {
 			needsHttpsSetup && {
 				key: 'force-secure-checkout',
 				title: __( 'Force secure checkout', 'woocommerce-payments' ),
-				content: __(
+				additionalInfo: __(
```

Comment:
We should keep this as `content` if we are setting `expandable` and `expanded` to true, given we render the `content` div when expandable is set to true, causing a weird gap between the additional info and empty content:
<img width="754" alt="Screen Shot 2021-09-15 at 10 22 32 AM" src="https://user-images.githubusercontent.com/2240960/133441385-26807151-10ef-4a03-971f-dbe6f2964bbb.png">

Given we are not using the `showActionButton` here, could we just set `expandable` and `expanded` to false and keep the `additionalInfo`?

Reply:
Nice catch @louwie17! This was fixed in the commit f2984f5.
As you mentioned, we should set `expandable` as `false` but since this is an expanded task, the prop `expanded` should be `true`.